frameworks/PHP/ubiquity/benchmark_config.json

```diff
@@ -18,10 +41,10 @@
         "flavor": "PHP7",
         "orm": "Full",
         "platform": "None",
-        "webserver": "nginx",
+        "webserver": "react",
         "os": "Linux",
         "database_os": "Linux",
-        "display_name": "ubiquity",
+        "display_name": "ubiquity-react",
         "notes": "",
         "versus": "php"
```

Comment:
Ideally will be versus `"react"`, but there isn't a vanilla react test.

So perhaps you can compare vs `"ubiquity"` and you will know the diference.
Will be more informative than vs "php".

What do you thing is the best @nbrady-techempower

Reply:
Hi @joanhey ,
Indeed, I asked myself the question.
versus php is not really satisfactory

Reply:
Like you said @joanhey, vs a vanilla react test would be idea, but totally up to you guys.

Reply:
To look at it more closely:
- hamlet swoole is vs php
- Laravel-swoole is vs swoole
Which is not very consistent.

Waiting for the vanilla React test,
what should I use: Ubiquity or php ?

Reply:
If I use `php` it allows me to see the performance of the `Ubiquity-React` couple compared to `PHP`.
If I use `Ubiquity`, it is to highlight React's performance
It seems to me that the comparison with PHP is more relevant for the moment

Reply:
I think that's fine, after all, you can look at the charts and compare it against anything you want.

Reply:
Thank you,
vanilla React test to come....